sys/auto_init/usb/auto_init_usb.c

```diff
@@ -33,6 +33,10 @@ usbus_cdcecm_device_t cdcecm;
 #ifdef MODULE_USBUS_CDC_ACM
 #include "usb/usbus/cdc/acm.h"
 #endif
+#ifdef MODULE_USBUS_DFU
+#include "usb/dfu.h"
+usbus_dfu_device_t dfu;
```

Comment:
```suggestion
static usbus_dfu_device_t dfu;
```

or better yet, move it together with `usbus_dfu_init`